.github/workflows/docker.yml

```diff
@@ -3,13 +3,14 @@ name: Publish Helper Image CI
 on:
   workflow_dispatch:
   push:
+    branches:
+      - master
     paths:
       - "docker/ci/**"
       - ".github/workflows/docker.yml"
-      - "Cargo.toml"
-  schedule:
-    # Once monthly at a randomly selected time.
```

Comment:
is it bad to keep running it this way?

Reply:
Bad how? The idea is to create a fresh new Docker on each merge, not once a month randomly. I envision this Docker going to a beta stage.

Reply:
My understanding is that this is complementary to each merge. If for some reason merges are not happening, it is good to keep generating images to catch regressions in the infrastructure

Reply:
So, this is just a start. I think in the future it might be better to tag each build with the commit_id AND with latest so that we can pin prod to a specific build and beta can just get the latest.

Reply:
when I start working on CI, we'll actually want to build every commit for PRs too. but we can sort that out down the road